sdks/python/apache_beam/io/hadoopfilesystem_test.py

```diff
@@ -517,6 +511,51 @@ def test_delete_error(self):
     self.assertFalse(self.fs.exists(url2))
 
 
+class HadoopFileSystemRuntimeValueProviderTest(unittest.TestCase):
+  """Tests pipeline_options, as passed via RuntimeValueProvider."""
```

Comment:
When does this get passed ? When running pipelines using templates ?

Reply:
I updated the comment. This test tests that options from RuntimeValueProvider.runtime_options are understood by HadoopFileSystem.